src/unitxt/catalog/augmentors/augment_whitespace_task_input.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
how will it look if its outside code?

Reply:
The augmentor object has two booleans "augment_model_input" and "augment_task_input".   Only one can be set (this is verified in the code).  The  standard template passes the needed input (model_input, ot task_input) to the augmentor , depending on these booleans.  The augmentor changes the relevant text.

In the catalog we save the augmentor , including the value of these flags, which determine it's behavior.

The users of the standard template, just calls with "augmentor=augmentors.augment_whitespace_model_input" or "augmentor=augmentors.augment_whitespace_model_input", and this creates the relevant augmentor based.